src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,9 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        Optional<User> optional = users.stream()
+                .filter(user -> user.getName().equals(name))
```

Comment:
```suggestion
                .filter(user -> user.sameName(name))
```
값을 꺼내와서 비교하기보단 객체에게 시켜보아요~